src/Application.php

```diff
@@ -125,6 +126,11 @@ private function registerCommands()
             ->get('application.commands.aliases')?:[];
 
         foreach ($consoleCommands as $name) {
+            // Some commands call AnnotationRegistry::reset, we need to ensure that
```

Comment:
As I said previously, while debugging, I discovered that the registration of some commands call `AnnotationRegistry::reset`. If we want to ensure, that the Annotation is correctly set, we must ensure it is correctly defined before the beginning of the registration of a new command.